components/salesforce_rest_api/actions/create-account/create-account.mjs

```diff
@@ -0,0 +1,49 @@
+import salesforce from "../../salesforce_rest_api.app.mjs";
+import account from "../../common/sobjects/account.mjs";
+import lodash from "lodash";
```

Comment:
Could you please destruct `lodash` and import only the function that you will need? I saw this behavior in some files, could you please destruct `lodash` in all them?